src/backend/tests/teams.test.ts

```diff
@@ -43,4 +45,36 @@ describe('Teams', () => {
     expect(prisma.team.findUnique).toHaveBeenCalledTimes(1);
     expect(prisma.team.findUnique).toHaveBeenCalledWith({ where: { teamId }, ...teamQueryArgs });
   });
+
+  describe('editTeam', () => {
+    test('setTeamMembers works', async () => {
+      jest.spyOn(prisma.team, 'findUnique').mockResolvedValue(prismaTeam1);
+      jest.spyOn(prisma.team, 'update').mockResolvedValue(prismaTeam1);
+      jest.spyOn(userUtils, 'getUsers').mockResolvedValue([superman, wonderwoman]);
```

Comment:
can we add a test case for this getUsers function where multiple aren't found and it throws the right stuff